src/writing-policies/rego/02-builtin-support.md

```diff
@@ -0,0 +1,102 @@
+# Builtin support
+
+Building a policy for the `wasm` target is only half of the problem,
+it needs to be executed.
+
+The Open Policy Agent team has a dedicated page you can check in order
+to [find out the built-in support
+level](https://www.openpolicyagent.org/docs/latest/policy-reference/#built-in-functions).
+
+Every green check in this table means that those built-ins are
+implemented regardless of the runtime: they are built on top of other
+functions and are implemented already on the policy you have built.
+
+The built-ins marked as `SDK-dependent` are the ones that the host has
+to implement -- in this case, Kubewarden. Open Policy Agent may use
+them in order to build the rest of the built-ins. In any case, this
+built-ins are exposed to the policy and any new or existing policy
+could depend on them.
+
+These are the built-ins implemented up until now in Kubewarden:
```

Comment:
I'm just a bit concerned about keeping this page up-to-date, but we can live with that :)

Reply:
I think it makes sense to not add this into the documentation. If you are good with it I can replace the explicit table with a link to https://github.com/kubewarden/policy-evaluator/issues/56, that is in our interest to keep up to date. WDYT?